pkg/domain/user/userRepositoryDb.go

```diff
@@ -0,0 +1,31 @@
+package user
+
+import (
+	"database/sql"
+)
+
+type UserRepositoryDb struct {
+	db *sql.DB
+}
+
+func (r UserRepositoryDb) Login(userObj User) (string,error){
+	row := r.db.QueryRow("select password from users where email=$1", userObj.Email)
+	err := row.Scan(&userObj.Password)
+	if err != nil {
+		return "",err
+	}
+	return userObj.Password,nil
+}
```

Comment:
how does this login work? uses just the email to return the password or what exactly?

Reply:
yes